src/pages/home/report/ReportActionCompose/SuggestionMention.js

```diff
@@ -296,16 +279,10 @@ SuggestionMention.propTypes = propTypes;
 SuggestionMention.defaultProps = defaultProps;
 SuggestionMention.displayName = 'SuggestionMention';
 
-export default withOnyx({
-    personalDetails: {
-        key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-    },
-})(
-    React.forwardRef((props, ref) => (
-        <SuggestionMention
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...props}
-            forwardedRef={ref}
-        />
-    )),
-);
+export default React.forwardRef((props, ref) => (
+    <SuggestionMention
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        forwardedRef={ref}
+    />
+));
```

Comment:
That'd require renaming an existing component logic, related to `forwardedRef` prop. Are we sure we want to add such change to the scope of this PR?

Reply:
Hi @aimane-chnaif! I'd love to change the syntax but I am afraid it might be out of scope of this PR, let me know if we could change that in separate PR or at least after reviewing most important changes. Thanks! 👍